### PR TITLE
Select Subscriptions icon

### LINKED ISSUE #351: Make selecting subscriptions more discoverable
from marketplace review: 

Rating: 5/5
Nice work! Please add some docs on how to filter the Subscriptions once you sign in with an Azure account. I discovered it after clicking on my account name on the lower part of the screen, but being a user with lots of subscriptions, it was important to know how to "clean up" the list.

### PR DESCRIPTION
**Pull request by StephenWeatherford**
Fixes #351

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,9 +29,21 @@
 - IntelliSense (completions) will be provided as you write your scripts
 - Select the script and press `CMD+"` (`CTRL+"` on Windows and Linux) to see the results
 
-## Sign in to Azure to view your Cosmos DB Accounts
+## Managing Azure Subscriptions
 
-![Azure Sign In](resources/signin.gif)
+Sign in to Azure to view your Cosmos DB Accounts
```
Comment (EricJizbaMSFT) [inline README.md]: Remove this line - looks like its leftover from the old header

File: package.json [hunk 2]
```diff
@@ -347,6 +355,11 @@
 				}
 			],
 			"view/item/context": [
+				{
+					"command": "cosmosDB.selectSubscriptions",
```
Comment (EricJizbaMSFT) [inline package.json]: Remember to exclude this from the command palette

Comment (StephenWeatherford) [reply package.json]: done

File: README.md (hunk 1 above)
Comment (StephenWeatherford) [reply README.md]: done

**Pull request merged**